rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -4308,12 +4287,12 @@ bool opcode_0184(const ScriptArguments& args, const ScriptCharacter character, c
     @arg vehicle Car/vehicle
     @arg arg2 
 */
-bool opcode_0185(const ScriptArguments& args, const ScriptVehicle vehicle, const ScriptInt arg2) {
+bool opcode_0185(const ScriptArguments& args, const ScriptVehicle vehicle, const ScriptInt value) {
     RW_UNIMPLEMENTED_OPCODE(0x0185);
     RW_UNUSED(vehicle);
-    RW_UNUSED(arg2);
+    RW_UNUSED(value);
     RW_UNUSED(args);
-    return true;
+    return false;
```

Comment:
Can you comment why here is false?

Reply:
Bad fix for taxi mission, because it always says "your car is trashed". This can be changed to check health with #448

Reply:
I'll revert this one in favor of the #448 implementation.